ColorPatterns/Scenes/StartMenu/StartMenuController.swift

```diff
@@ -24,6 +24,8 @@ final class StartMenuController: UIViewController {
     }
     
     @IBAction private func difficultyButtonPressed(_ sender: UIButton) {
+        // TODO: it's not a good idea to use tags to do such things I'd go with enum difficulty: easy, med, hard and setup buttons with these values, but not use tags
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/vandermesis/ColorPatterns/blob/c07cd9a3b7a63e61e86332aeb4fa4cabf8b81c17/ColorPatterns/Scenes/StartMenu/StartMenuController.swift#L27"></span>TODOs should be resolved (it’s not a good idea to use ta…).<br />
<code>todo</code> <code>StartMenuController.swift:27</code></td>
    </tr>
  </tbody>
</table>